examples/vapor_example/BUILD.bazel

```diff
@@ -23,7 +23,7 @@ gazelle_binary(
 gazelle(
     name = "update_build_files",
     args = [
-        "-swift_dependency_index=swift/deps_index.json",
+        "-swift_dependency_index=swift/swift_deps_index.json",
```

Comment:
Do we need to rename the file to be `swift_deps_index.json`? The original intent was to allow the client to place the file where they wanted with whatever name they wanted.

Reply:
ah I see the default is `swift_deps_index`, i thought i was changing the name to validate both the name being custom and the path but i'll revert this to `deps_index` since thats custom